config/server.lua

```diff
@@ -127,7 +127,7 @@ return {
     ---@param player Player Player object
     ---@param payment number Payment amount
     sendPaycheck = function (player, payment)
-        player.Functions.AddMoney('bank', payment)
+        AddMoney(player.PlayerData.source, 'bank', payment)
```

Comment:
This needs to be an export call, no? Since it will be invoked from within another script, the AddMoney function won't exist in the other script.